src/rgw/rgw_lua.cc

```diff
@@ -238,6 +238,9 @@ int install_packages(const DoutPrefixProvider *dpp, rgw::sal::Driver* driver,
     ldpp_dout(dpp, 1) << "Lua ERROR: failed to create temporary directory from template: " << 
       tmp_path_template << ". error: " << rc << dendl;
     return rc;
+  } else {
+    // rgw starts as root and will later drop to user ceph
+    umask(tmp_luarocks_path, 0755)
```

Comment:
Missing semicolon prevents this from compiling. Please compile and do your own testing before adding the `needs-qa` label.